client/client.go

```diff
@@ -0,0 +1,46 @@
+package client
+
+import (
+	"net/url"
+	"sync"
+
+	cclient "github.com/deis/workflow-cli/controller/client"
+	"github.com/goware/urlx"
+)
+
+type Client struct {
+	rwm *sync.RWMutex
+	cl  *cclient.Client
```

Comment:
perhaps something less confusing like `deisClient` would be more helpful when grokking code, but this is a small nitpick.